docs/cloud/sso.md

```diff
@@ -123,7 +123,10 @@ Steps:
 4. Under Configure platforms, select the web option.
 5. Select Configure to complete the platform configuration.
 
-Once you finish registering lakeFS Cloud with Azure AD, save the **Application (Client) ID**, **Application Secret Value** and send this to Treeverse's team to finish the integration.
+Once you finish registering lakeFS Cloud with Azure AD, send the following values to the Treeverse team:
```

Comment:
- Can we add example values?
- Do we need to specify how? like the email - if there are multiple ways, leave it.